### PR TITLE
#1719 - Create Release Strategy 

### PR DESCRIPTION
**Pull request by ann-aot**
- added `gitRef` for all checkouts
- removed default  `main` from all workflow dispatch input for  `gitRef/buildRef`

### CODE REVIEW

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @ann-aot

File: .github/workflows/deploy-all.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: Deploy to OpenShift/Camunda
 on:
   workflow_dispatch:
     inputs:
-      buildRef:
-        description: "Build Ref"
+      gitRef:
```
Comment (andrewsignori-aot) [inline .github/workflows/deploy-all.yml:6]: I believe that the `default: "main"` does not make much sense anymore.
We can leave it blank and maybe provide a description like `Git Ref (e.g, v1.0.0 or v1.0.0-prerelease.1)`.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the change, it looks good 👍

**Review by andrepestana-aot — APPROVED** [review]
LGTM!

**Pull request merged**